app/src/main/java/com/hedvig/app/util/extensions/ActivityExtensions.kt

```diff
@@ -4,31 +4,54 @@ import android.app.Activity
 import android.content.Intent
 import android.content.pm.PackageManager
 import android.net.Uri
+import android.os.Build
 import android.provider.Settings
-import android.util.DisplayMetrics
 import android.view.View
+import android.view.WindowInsets
 import androidx.annotation.DrawableRes
 import androidx.annotation.IdRes
 import androidx.appcompat.app.AppCompatActivity
 import androidx.appcompat.widget.Toolbar
 import androidx.core.app.ActivityCompat
 import androidx.core.app.ActivityOptionsCompat
+import androidx.core.view.WindowInsetsCompat
 import com.hedvig.app.R
 import com.hedvig.app.authenticate.AuthenticateDialog
 import com.hedvig.app.feature.chat.ui.ChatActivity
 import com.hedvig.app.feature.offer.ui.OfferActivity
 import com.hedvig.app.util.extensions.view.setupToolbar
 import e
 
-val Activity.displayMetrics: DisplayMetrics
+val Activity.screenWidth: Int
+    get() = window.decorView.measuredWidth
```

Comment:
Is this correct? 🤔

Reply:
I haven't changed this (just moved) so I suppose its correct